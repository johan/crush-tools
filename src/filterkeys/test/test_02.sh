test_number=02
description="one key filter field 1 (inverse)"

outfile="$test_dir/test_$test_number.actual"
expected="$test_dir/test_$test_number.expected"

$bin -p -v -a 1 -b 1 -f "$test_dir/test-filter.in" "$test_dir/test-1.in" > "$outfile"

if [ $? -ne 0 ] ||
   [ "`diff -q $outfile $expected`" ]; then
  test_status $test_number 1 "$description" FAIL
else
  test_status $test_number 1 "$description" PASS
  rm "$outfile"
fi
