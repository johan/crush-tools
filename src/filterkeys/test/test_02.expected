key1^key2^value1^value2
xdx^aaa^1234^98.765
nhf^aab^1234^56.341
qop^aab^3491^56.341
lrf^erb^3491^56.341
xdb^awz^3491^98.765
nym^awz^3491^98.765
rti^erb^3491^98.765
adp^aaa^1234^98.765
